package.json

```diff
@@ -8,7 +8,7 @@
     "test": "cross-env NODE_ENV=test mocha --recursive -r ts-node/register src/**/*.spec.ts --exit",
     "test:validation": "cross-env NODE_ENV=test mocha --recursive -r ts-node/register \"src/**/*.validator.spec.ts\" --exit",
     "test:integration": "cross-env NODE_ENV=test mocha --recursive -r ts-node/register \"src/**/*.router.spec.ts\" --exit",
-    "test:repository": "cross-env NODE_ENV=test mocha --recursive -r ts-node/register \"src/**/*.repository.spec.ts\" --exit",
+    "test:repository": "cross-env NODE_ENV=test mocha --recursive --timeout 5000 -r ts-node/register \"src/**/*.repository.spec.ts\" --exit",
```

Comment:
No external services are used here. No need to change default timeout. If you get timeout errors, there is something wrong with the implementation

Reply:
Done.